opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/Adis.java

```diff
@@ -983,8 +983,16 @@ public ReservationResult reservation(DetailedItem item, Account account,
                         }
                     }
 
-                    if (doc.select(".message h1").size() > 0) {
-                        String msg = doc.select(".message h1").text().trim();
+                    String msgTest = "";
+                    if ( doc.select(".message h1").size() > 0){
```

Comment:
Could you please reformat the code (Ctrl+Alt+L in Android Studio) to conform to the overall style (e.g. no space after `(`, but space after `)`, no line break before `else if`, etc.)?